### ISSUE TITLE
test and npy

### ISSUE THREAD

**Issue by 30491** [qilong-zhang/Pytorch_Universal-adversarial-perturbation#4]
I can't get the same result from your code, I think maybe my test method is different from yours.
Can you upload the test code and the npy file (googlenet) of your experiment, especially how to add the npy file to the picture.

**Comment by qilong-zhang** [comment]
Hi @30491, do you load GoogleNet by `torch.nn.Sequential(Normalize(mean, std), models.googlenet(pretrained=True).eval()).cuda()`?

**Comment by 30491** [comment]
Yes, I do, and my success rate is only about 60%

**Comment by qilong-zhang** [comment]
Do you run 5 epochs? Besides, the testing model in my default code is Inception_v3, and the success rate on it is about 60%.

**Comment by 30491** [comment]
sorry， i only ran 3 epochs and i can't find where the Inception_v3 is. 
It would be greatly appreciated if ‘npy’ files could be provided

**Comment by qilong-zhang** [comment]
I load inception_v3 in https://github.com/qilong-zhang/Pytorch_Universal-adversarial-perturbation/blob/85fb4c2236f6e9020dfe33838c1888a7e7895e7a/search.py#L20

I am a little confused about the `npy` file you mentioned. It should be the same if you also load GoogleNet from `torchvision.models`

**Comment by 30491** [comment]
sorry，I changed inception_v3 to googlenet. I forgot it. I will re-experiment.
Thanks！
